frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -714,6 +691,7 @@ pub mod pallet {
 			AccountId = Self::AccountId,
 			BlockNumber = Self::BlockNumber,
 			DataProvider = Self::DataProvider,
+			MaxWinners = Self::MaxWinners,
```

Comment:
As you see, all election provider are now bounded by default, at the type level. While we did discuss that this is a bit more restrictive, it does make life easier by making everything have a unified `Get<u32>` bound across the board.